crates/cxx-qt-gen/test_inputs/properties.rs

```diff
@@ -20,4 +20,14 @@ mod ffi {
         #[qproperty(i32, final_prop, READ, WRITE, FINAL)]
         type MyObject = super::MyObjectRust;
     }
+
+    unsafe extern "RustQt" {
+        fn my_getter(self: &MyObject) -> i32;
+
+        fn my_setter(self: Pin<&mut MyObject>, value: i32);
```

Comment:
Maybe add some `#[cxx_name]` or `#[rust_name]` to some of these to test the renaming actually works with custom renaming.

Reply:
Done